server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
+Die soort is geëvolueerd.
+Charles Darwin is de grondlegger van de evolutietheorie.
+De politieagent was op zoek naar een cruciaal stukje bewijs.
+Ze hadden hem dan toch tot een bekentenis kunnen dwingen.
+Folteren is verboden in België.
+Naar verluidt worden er mensen gefolterd in Guantanamo, een Amerikaanse gevangenis op Cuba.
+De Chinese regering heeft het voorstel van de Russische regering geweigerd.
+De ingenieurs zijn druk bezig om een nieuwe brug te bouwen.
+Dat is een ingenieus systeem.
+Er staat een barcode op de achterzijde.
+De stoel is nogal hard dus heb ik er een kussentje op gelegd.
+De lamp is dimbaar.
+Kan je de kleur veranderen?
+Ik zie graag je roze muur.
+Ik heb de kast beige geverfd.
+De was is klaar.
+Prima, dan steek ik het in de droogkast.
+Een flinke roman telt tienduizend zinnen.
+Hou je de statistieken bij?
+De schijnwerper was op haar gericht.
+Ken jij de ingrediënten van een mojito?
+Neen, maar ik heb liever een caipirinha.
+Een gin en tonic kan ook verfrissend zijn.
+Ik moet er nog citroensap aan toevoegen.
+Doe mij maar een kirr royal.
+Oh, ik heb wel zin in een aprerol spritz.
+Ken je nog cocktails?
+Ik heb een hongertje.
+Zouden we wat nootjes mogen bij de drank?
+Ik stuur je een mailtje via Gmail.
+Het zachte briesje maakte me terug wakker.
+Speel "Rood" van Marco Borsato.
+Speel "Sterrenstof" van de Jeugd van Tegenwoordig.
+Speel "Bagagedrager" van Gers Pardoel.
+Kan je "Ik neem je mee" van Gers Pardoel afspelen?
+De oranje gloed van de straatverlichting verlichte de kamer.
+Louise is een gymnaste.
+Gymnastiek is zijn specialiteit.
+Er kroop een mier over mijn been.
+Ze waren met hun huurauto frontaal op een termietenheuvel gebotst.
+De mug had me de hele nacht wakker gehouden.
+Hij heeft haar in zijn macht.
+De machtige heerser leidde al tien jaar het land.
+Er zat een barst in het glas.
+Zonder de bliksemafleider was de toren al lang afgebrand.
+De bliksemschicht was te zien aan de horizon.
+Hij had hen alles verteld onder hypnose.
+Ik heb de hik.
+Hij moest niezen.
+Ze bekeek hem met argwaan.
+Ik vind geen kapstok.
+Nee dat denk ik niet.
+Voor mij een whisky met ijs.
+Je kan dat wel vinden op Wikipedia.
+We gaan naar Schotland om te wandelen.
+Guinness word gebrouwen in Ierland.
+Kan je de route even opzoeken op Google Maps?
+Stella Artois is een pils die wordt gebrouwen in Leuven.
+Ik lust wel een Duvel.
+Ik voelde een druppel.
+Mijn nagels waren te lang.
+Ik ga verhuizen naar het buitenland.
+Ben je bang van spinnen?
+Pythons zijn één van de families van slangen.
+Het jachtluipaard liep over de vlakte.
+Machu Picchu is een oude stad in Peru.
+Met dit weer heb ik meer last van astma.
+Neem de eerste afslag op de rotonde.
+Aan het kruispunt moet je naar rechts afslaan.
+De auto was in panne gevallen op de oprit van de autostrade.
+Er is een tunnel voor fietsers.
+Neem de eerste afslag na de brug.
+Vanaf hier is het altijd rechtdoor.
+Het is de tweede afslag op het rond punt.
```

Comment:
"het rond punt", is dit correct? zou "de ronde punt" verwachten  => de rotonde (BE/NL?)